src/js/components/form/FieldAutofocus.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import DOMUtils from "#SRC/js/utils/DOMUtils";
+
+let lockFieldAutofocus = false;
+const SUPPORTED_TYPES = ["FieldInput", "FieldTextarea", "input", "textarea"];
+
+class FieldAutofocus extends React.Component {
+  componentDidMount() {
+    if (lockFieldAutofocus) {
+      return;
+    }
+    const input = DOMUtils.getInputElement(ReactDOM.findDOMNode(this));
+    if (!input) {
+      return;
+    }
+    lockFieldAutofocus = true;
+    input.focus();
+
+    input.addEventListener("blur", function onBlurInput() {
+      lockFieldAutofocus = false;
+      this.removeEventListener("blur", onBlurInput);
+    });
+  }
+
+  render() {
+    return this.props.children;
+  }
+}
+
+FieldAutofocus.propTypes = {
+  children(props, propName, componentName) {
```

Comment:
I believe we can remove this, since we are moving away fro propTypes and this is a custom one so I vote to remove it.